ldap/resource_ldap_group.go

```diff
@@ -25,6 +25,11 @@ func resourceLDAPGroup() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Is it necessary to force the recreation of the group on description update ?

Reply:
nope. I started something for an in place update, but I didn't find a way to remove the description attribute when it's removed from the terraform resource. Any suggestion for that ?

Reply:
Done in https://github.com/Ouest-France/terraform-provider-ldap/pull/6/commits/d772dc86ff55219e1f3a93839f56eedcf055c393

Reply:
@pablo-ruth any suggestion for this ?

```go
func (c *Client) UpdateGroup(dn, name string, description string) error {

	req := ldap.NewModifyRequest(dn, []ldap.Control{})

	if description != "" {
		req.Replace("description", []string{description})
	}

	return c.Conn.Modify(req)
}
```

This work for description updates, but I can't delete the description ldap attribute when the argument is not defined in Terraform.

Reply:
You can delete the "description" attribute when empty:
```golang
func (c *Client) UpdateGroup(dn, name string, description string) error {

	req := ldap.NewModifyRequest(dn, []ldap.Control{})

	if description == "" {
		req.Delete("description", []string{})
	} else {
		req.Replace("description", []string{description})
	}

	return c.Conn.Modify(req)
}
```

Reply:
Never mind. Fixed in https://github.com/Ouest-France/terraform-provider-ldap/pull/6/commits/2a62d62d8ef398ef8533a286ea40786a57f25c94